classes/phing/system/io/FileSystem.php

```diff
@@ -293,13 +298,18 @@ public function getLastModifiedTime(PhingFile $f)
      */
     public function getLength(PhingFile $f)
     {
+        if (PHP_MAJOR_VERSION >= 7) {
```

Comment:
I think we could drop all occurrences of the `PHP_MAJOR_VERSION >= 7` conditions as the minimum php version requirement for phing is PHP >= 7

Reply:
I've dropped that check on PHP_MAJOR_VERSION now.